Sources/MapboxDirections/IsochroneOptions.swift

```diff
@@ -0,0 +1,302 @@
+import Foundation
+import Turf
+
+#if canImport(UIKit)
+import UIKit
+#elseif canImport(AppKit)
+import AppKit
+#endif
+
+
+/**
+ Options for calculating contours from the Mapbox Isochrone service.
+*/
+public class IsochroneOptions {
+    
+    public init(centerCoordinate: LocationCoordinate2D, contours: Contours, profileIdentifier: ProfileIdentifier = .automobile) {
+        self.centerCoordinate = centerCoordinate
+        self.contours = contours
+        self.profileIdentifier = profileIdentifier
+    }
+    
+    // MARK: Configuring the Contour
+    
+    /**
+     Contours GeoJSON format.
+     */
+    public enum ContourFormat {
```

Comment:
Do we need to nest this type under `IsochroneOptions`? Application code might end up needing to write `IsochroneOptions.ContourFormat` in some cases, but `ContourFormat` on its own would probably not risk future conflicts with other API wrappers in this library.

Reply:
I've nested this one since there's no practical use of this enum other than inside `IsochroneOptions`. Cases when fully qualified name would be required for typing are pretty rare (Linux only?) since there is only 1 place (property) where it is used and it's strictly typed.